src/main/assembly/dist/bin/easy-pid-generator

```diff
@@ -34,27 +35,21 @@ start_jsvc_exec()
 {
     echo "Starting $NAME ..."
     jsvc_exec
-
-    local exitcode=$?
-
-    if [[ $exitcode == 0 ]]; then
+    if [[ $? == 0 ]]; then
```

Comment:
Sure. Then it's all well, I suppose! Gonna merge this into my PR. Then I can also add this to the other projects.